pkg/iris/api.go

```diff
@@ -70,3 +71,30 @@ func (a *api) PostResults(results []byte, accessToken string) error {
 	log.Infof("Event posted successfully [%s]", resp.Status)
 	return nil
 }
+
+func (a *api) PostStatus(status []byte, accessToken string) error {
```

Comment:
Just for understanding. Does post status should be different from post results? I understood post result should be generic to use.